src/modules/app/routes/pointsOfInterest.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+
+import { createAction } from 'redux-actions'
+import type { Dispatch, GetState } from 'redux-first-router'
+import pointsOfInterestEndpoint from '../../endpoint/endpoints/pointsOfInterest'
+
+export const POINTS_OF_INTEREST_ROUTE = 'POINTS_OF_INTEREST'
+
+export const goToPointsOfInterest = (city: string, language: string, pointOfInterestId: ?string) =>
+  createAction(POINTS_OF_INTEREST_ROUTE)({city, language, pointOfInterestId})
+
+export const getPointsOfInterestPath = (city: string, language: string, pointOfInterestId: ?string): string =>
+  `/${city}/${language}/poi${pointOfInterestId ? `/${pointOfInterestId}` : ''}`
```

Comment:
Hmm... It might be easier to use /locations instead of /poi -- also we use plural for extras and events, so we should use plural here as well -- : The cms will probably create (internal) links pointing to sth. with /locations as path segment, so e.g. 
https://cms.integreat-app.de/augsburg/de/locations/cafe-tuer-an-tuer (is this correct, @timoludwig / @svenseeberg ?) 
It would the easiest for us, if we just adopted this I guess (when it comes to creating sitemaps, internal navigation in webapp, etc. pp.)

Reply:
correct, the permalinks of pois will always have `locations` in it, no matter what we call the api endpoint.

Reply:
We have so many locations already :/ LocationLayout, location of the router, cities have been locations at some time. Of course, I don't really have a problem with locations.

Reply:
Same question again :stuck_out_tongue:

Reply:
@maxammann opinion required

Reply:
Yes, lets go with locations if we can not change the permalinks.
But I think we should use the name poi in the client. So the we never really have to care that its named location in the backend.

Reply:
So in this example I would use the name poi.js for the route.

Reply:
So our route should be /augsburg/de/pois or /augsburg/de/locations? And the name of the files and routes should be poi.js?

Reply:
Yes the route with locations and the files with pois

Reply:
@timoludwig could you maybe supply the permalink id as a separate attribute?  i.e. 'cafe-tuer-an-tuer' in '/augsburg/de/locations/cafe-tuer-an-tuer/'?

Reply:
@maxammann @Schedulaar sollten wir dann LocationLayout und so auch mal zu CitiesLayout umbenennen? Oder lassen wir das so wies ist?

Reply:
@timoludwig wir brauchen die permlink id dann auch in den available_languages

Reply:
@klinzo Afaik ist https://cms.integreat-app.de/augsburg/en/wp-json/extensions/v3/locations fast genauso aufgebaut wie der pages endpoint - insbesondere der available_languages bereich ist identisch... Sollte also also eigentlich schon gehen oder? :thinking:

Reply:
yes, no problem. i will add the ´permalinkId´ property to all post types (pages, events, locations, disclaimer).

Reply:
@Schedulaar Ja schon, der Unterschied is nur, dass wir für Pages den kompletten path (`/augsburg/de/willkommen/erste-schritte`) verwenden, weil wir ja mehrere Ebenen haben. Das macht bei pois und events aber keinen Sinn, da reicht die pathId, zb `cafe-tür-an-tür`. Die steckt zwar im Pfad auch drin, is aber in der WebApp schöner, wenn wir die extra geliefert bekommen anstatt die aus dem path raussuchen  zu müssen. Also ja geht, is aber hässlich :D

Reply:
Hatte nen Denkfehler, dass wir die permlink Id brauchen, um den path zu bekommen, unter dem der poi oder das event aufrufbar ist... Aber den bekommen wir ja schon vom cms :man_facepalming: :unamused:

Reply:
@klinzo About the layout names. Both names are fine imo. But I'd leave it as it is until we gain benefit from a change.